### PR TITLE
Feature/todos

### PR DESCRIPTION
**Pull request by RomanSoldatov**

### CODE REVIEW

File: controllers/boardsControllers.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+const { Board } = require("../schemas/boardsSchemas");
+const { Column } = require("../schemas/columnsSchemas");
+const { Todo } = require("../schemas/todosSchemas");
+
+const getAllBoards = async (req, res) => {
+  const { _id: owner } = req.user;
+  const { page = 1, limit = 10 } = req.query;
+  const boards = await serv.listBoards({ owner }, page, limit);
+  res.status(200).json(boards);
+};
```
Comment (DmytroPerepelytsiaa) [inline controllers/boardsControllers.js]: Do we need this pagination?

File: controllers/columnsControllers.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+//   const { page = 1, limit = 10 } = req.query;
+//   const columns = await serv.listColumns({ owner }, page, limit);
+//   res.status(200).json(columns);
+// };
+
+// const getById = async (req, res, next) => {
+//   const { columnId } = req.params;
+//   const column = await serv.getColumnById(columnId);
+//   res.status(200).json(column);
+// };
```
Comment (DmytroPerepelytsiaa) [inline controllers/columnsControllers.js]: if you don't need this - delete, don't push it to github

File: controllers/columnsControllers.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+};
+
+const updateColumn = async (req, res) => {
+  const { id } = req.params;
+  const column = await serv.updateColumn(id, req.body);
+  res.status(200).json(column);
+};
+
+module.exports = {
+  // getAllColumns: ctrlWrapper(getAllColumns),
```
Comment (DmytroPerepelytsiaa) [inline controllers/columnsControllers.js]: same

File: controllers/todosControllers.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+// const getAllTodos = async (req, res) => {
+//   const { columnId: owner } = req.params;
+//   const { page = 1, limit = 10, priority } = req.query;
+//   const todos = await serv.listTodos({ owner }, page, limit, priority);
+//   res.status(200).json(todos);
+// };
+
+const getById = async (req, res) => {
+  const { id } = req.params;
+  console.log(id);
```
Comment (DmytroPerepelytsiaa) [inline controllers/todosControllers.js]: don't push console logs

File: middlewares/isCurrentUserOwner.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+const { HttpError } = require("../helpers");
+const serv = require("../services/boardsServices");
+
+const isCurrentUserOwner = async (req, res, next) => {
+  const { id } = req.params;
```
Comment (DmytroPerepelytsiaa) [inline middlewares/isCurrentUserOwner.js]: Better to name it like isUserBoardOwner

Comment (DmytroPerepelytsiaa) [reply middlewares/isCurrentUserOwner.js]: and filename too

File: routes/boardsRouter.js [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+boardsRouter.post(
+  "/",
+  validateBody(schemas.createBoardSchema),
+  authenticate,
+  ctrl.createBoard
+);
+
+boardsRouter.put(
+  "/:boardId",
+  isValidId,
```
Comment (DmytroPerepelytsiaa) [inline routes/boardsRouter.js]: We really need put methods if we already have patch?

File: schemas/boardsSchemas.js [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+const boardSchema = new Schema(
+  {
+    title: {
+      type: String,
+      required: true,
+      //   maxLength: 30,
+      trim: true,
+    },
+    iconsURL: String,
+    backgroundURL: String,
```
Comment (DmytroPerepelytsiaa) [inline schemas/boardsSchemas.js:12]: better to name it iconURL, because you store only 1 icon

**Review by DmytroPerepelytsiaa — CHANGES_REQUESTED** [review]

File: controllers/boardsControllers.js (hunk 1 above)
Comment (RomanSoldatov) [reply controllers/boardsControllers.js]: obviously not, will remove

File: controllers/columnsControllers.js (hunk 2 above)
Comment (RomanSoldatov) [reply controllers/columnsControllers.js]: I wasn't going to push this, I wanted to get working code and then clean everything up

File: routes/boardsRouter.js (hunk 6 above)
Comment (RomanSoldatov) [reply routes/boardsRouter.js]: may be dont....

**Review by DmytroPerepelytsiaa — APPROVED** [review]

**Pull request merged**